spring-cloud-alibaba-nacos-config/src/main/java/com/alibaba/cloud/nacos/NacosConfigProperties.java

```diff
@@ -81,6 +85,26 @@
 	 */
 	private int timeout = 3000;
 
+	/**
+	* nacos maximum number of tolerable server reconnection errors
```

Comment:
comment should be end with "."

Reply:
![image](https://user-images.githubusercontent.com/36881032/62523611-c1da6080-b866-11e9-83ac-cf4de8bdb8e4.png)
What's wrong with this, please

Reply:
I don't know. If that's the case tomorrow, I suggest turning off PR and creating a new one.

Reply:
Do I need to download this PDF file and send it to the designated email address?

Reply:
only need authorization at the buttom